### DISCUSSION TITLE
Change name for module Grid

### DISCUSSION THREAD

**Discussion by seaj1one** [builderius/builderius#44] [Ideas]
I'm suggesting a name change because it could be confused with CSS Grid. I initially thought that. I was thinking of Dynamic table or  Dynamic table builder. 

Does anyone have any thoughts?

![image](https://user-images.githubusercontent.com/72103864/200041178-d69fa058-6e35-4177-be3d-c8b7ac9b7890.png)

**Comment by mrpsiho** [comment]
Agree. It should be table grid, probably.

**Comment by seaj1one** [comment]
Are you ok to add it to the roadmap for us please from here or should I @mrpsiho ?

**Reply by mrpsiho** [reply]
It is here now: https://github.com/builderius/builderius/issues/46 and it is a part of our milestone https://github.com/builderius/builderius/milestone/2

**Comment by mrpsiho** [comment]
Not relevant.
